docs/maven-plugin/src/main/java/org/keycloak/guides/maven/Guide.java

```diff
@@ -47,4 +48,12 @@ public int getPriority() {
     public void setPriority(int priority) {
         this.priority = priority;
     }
+
+    public Boolean getTileVisible() {
```

Comment:
Any reason why we're not using `boolean`? From the current usage it seems that we don't need the boxing. We can then just do `if (!guide.getTileVisible() && guide.getPriority() == Integer.MAX_VALUE) ...` in `Context.java`

Reply:
I thought I might need a tri-state value, but you are right that this can be simplified to a `boolean` after setting its default value to `true`. 

Changes applied in next commit.